index.bikeshed.bs

```diff
@@ -36,22 +36,47 @@ spec: ABNF; urlPrefix: https://tools.ietf.org/html/rfc5234
     text: VCHAR; url: appendix-B.1
     text: WSP; url: appendix-B.1
 
+spec: CSP; urlPrefix: https://w3c.github.io/webappsec-csp/
+  type: dfn
+    text: Content Security Policy; urlPrefix: #
+    text: policy; url: policy
+    text: directive; url: directives
+    text: directive value; url: directive-value
+    text: pre-request check; url: directive-pre-request-check
+    text: create a violation object for global; url: create-violation-for-global
+    text: report violation; url: report-violation
+    text: violation; url: violation
+    text: violation-resource; url: violation-resource
```

Comment:
Ditto here: `text: resource; for: violation; url: violation-resource`. Linked as `<a for="violation">resource</a>`.